cmd/podman/early_init_linux.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+func setRLimits() error {
+	rlimits := new(syscall.Rlimit)
+	rlimits.Cur = 1048576
```

Comment:
Nit: Can we make this a constant in `define`? It's also being used in `pkg/spec`.

Reply:
yes that is a good idea.  Do we know where we took this magic number and what it means?

Reply:
called `define.RLimitDefaultValue` for now